### ISSUE TITLE
Proper implementation of RecoveryAccessor::postWrite()

### ISSUE THREAD

**Issue by killenb** [ChimeraTK/ApplicationCore#139]
Child of #123 
Depends on #130 

Currently doPostWrite is calling the generic transfer. This does not make sense with the new scheme.
* The open-sequence has already been removed in #130 
* SetOwnerValidity is only there for write operations and does not apply here

Correct implementation:
`postReadWithoutErrorBlocking()`:
* Try calling NDRegisterAccessorDecorator::doPostWrite() and catch any excetion. Store it and set hasSeenException to true.
* If hasSeenException report it. 

`doPostRead()`:
* postReadWithoutErrorBlocking();
* if (hasSeenException) deviceModule->waitForRecovery();

There is no while loop which re-tries the action. The recovery accessor has already done the job when waitForRecovery returns.

**Comment by killenb** [comment]
This ticket was too small anyway and is now obsolete.  will be done in #130 because it turned out the generic transfer does not work anyway
